paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/FormViewModelModule.kt

```diff
@@ -1,15 +1,8 @@
 package com.stripe.android.paymentsheet.injection
 
-import com.stripe.android.ui.core.forms.resources.AsyncResourceRepository
-import com.stripe.android.ui.core.forms.resources.ResourceRepository
-import dagger.Binds
 import dagger.Module
 
 @Module(
     subcomponents = [FormViewModelSubcomponent::class]
 )
-internal abstract class FormViewModelModule {
-    @Binds
-    abstract fun bindsResourceRepository(asyncResourceRepository: AsyncResourceRepository):
-        ResourceRepository
-}
+internal abstract class FormViewModelModule
```

Comment:
If this is empty it can be deleted, and you can make `FormViewModelSubcomponent` a subcomponent of `FormViewModelComponent` by making it a val in `FormViewModelComponent`, see 1 [here](https://medium.com/mobile-app-development-publication/dagger-2-subcomponent-through-module-or-component-comparison-92486767a861)